.github/workflows/tests.yml

```diff
@@ -0,0 +1,185 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  linux:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php: ["5.6", "7.0", "7.1", "7.2", "7.3", "7.4"]
+        stability: ["Stable"]
```

Comment:
What is the profit of using `stability` and the then triplicated "Install dependencies" with if?

Why can't we have key like `composer_flags: '--prefer-lowest'` and use it in the command?

Reply:
Because this way we can easily render what is going on in the GitHub Actions UI for job descriptions. Much more important that those are readable, than the config file is.